packages/web/src/components/organisms/EntityView/EntityView.tsx

```diff
@@ -175,8 +168,74 @@ const EntityListContent: React.FC<EntityListBlueprint> = props => {
         hasMore={!isLoadingNext && data && queryFilters.pageSize <= data.length}
         onScrollEnd={onScrollBottom}
       />
+    ),
+    [
+      CardComponent,
+      EmptyData,
+      data,
+      isFiltersEmpty,
+      isFirstTimeLoading,
+      isLoadingNext,
+      isWritable,
+      itemKey,
+      onAdd,
+      onItemAbort,
+      onItemClick,
+      onScrollBottom,
+      queryFilters.pageSize,
+      resetFilters,
+    ]
+  );
+
+  const viewComponent = useTestsSlotFirst('entityViewComponent', [{value: entityGrid, metadata: {order: 2}}]);
+
+  return (
+    <PageWrapper>
+      <PageMetadata title={pageTitle} />
+
+      <PageHeader
+        title={pageTitle}
+        pageTitleAddon={pageTitleAddon}
+        description={PageDescription}
+        loading={isApplyingFilters && !isFirstTimeLoading}
+      >
+        <PageToolbar extra={createButton}>
+          <FiltersSection>
+            <EntityViewFilters
+              setFilters={setQueryFilters}
+              filters={queryFilters}
+              disabled={isEmptyData || !isReadable}
+            />
+          </FiltersSection>
+        </PageToolbar>
+      </PageHeader>
+
+      {isValidElement(viewComponent)
+        ? cloneElement(viewComponent, {
+            data,
+            loadingInitially: isFirstTimeLoading,
+            empty: isFiltersEmpty ? (
+              <EmptyData action={onAdd} isClusterAvailable={isWritable} />
+            ) : (
+              <EmptyDataWithFilters resetFilters={resetFilters} />
+            ),
+            hasMore: !isLoadingNext && data && queryFilters.pageSize <= data.length,
+            loadingMore: isLoadingNext,
+            onScrollEnd: onScrollBottom,
+          } as Partial<unknown>)
+        : null}
     </PageWrapper>
   );
 };
 
-export default memo(EntityListContent);
+export default EntityView;
+
+// Styled Components
+
+const FiltersSection = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 16px;
+  flex-wrap: wrap;
+`;
```

Comment:
Discussion: in case of such 1-2 styled components:
1. Should we still keep it in `.styled.ts` file
2. Should we put it on the end of file like here
3. Should we put it on top of the file

@topliceanurazvan, @devcatalin, @haneabogdan

Reply:
I think we should go with the first option, at least that pattern should be followed in this PR, just have to fix this comment here

Reply:
I would still keep the `.styled.ts` file for consistency